shoov/modules/custom/shoov_restful/plugins/restful/user/config/1.0/ShoovConfigResource.class.php

```diff
@@ -26,19 +26,13 @@ public function publicFieldsInfo() {
       'callback' => array($this, 'getAccessToken'),
     );
 
-    $public_fields['browserstack_username'] = array(
-      'property' => 'field_browserstack_username',
-    );
-
-    $public_fields['browserstack_key'] = array(
-      'property' => 'field_browserstack_key',
-    );
-
     unset($public_fields['id']);
     unset($public_fields['self']);
     unset($public_fields['label']);
     unset($public_fields['url']);
     unset($public_fields['mail']);
+    unset($public_fields['browserstack_username']);
```

Comment:
those fields need to still be kept here - we might have a follow up to merge `Config` and `Account` together